org.eclipse.zest.core/src/org/eclipse/zest/core/widgets/GraphContainer.java

```diff
@@ -176,13 +177,9 @@ public void close(boolean animate) {
 		updateFigureForModel(getModelFigure());
 	}
 
-	private static void addNodeToOrderedList(List orderedNodeList, GraphNode node) {
-		Iterator orderedNodeIterator = orderedNodeList.iterator();
+	private static void addNodeToOrderedList(List<GraphNode> orderedNodeList, GraphNode node) {
 		int counter = 0;
-		while (orderedNodeIterator.hasNext()) {
-			// Look through the list of nodes below and find the right spot for
-			// this
```

Comment:
should we preserve this comment?

Reply:
Good point. Seems like that comment was removed by the auto-formatter. I've restored it.